app/controllers/v6/assessments_controller.rb

```diff
@@ -211,6 +214,17 @@ def parse_outgoings(outgoings_params)
       }.flatten
     end
 
+    def parse_other_incomes(other_incomes_params)
+      other_incomes_params.map { |other_income|
+        other_income[:payments].map do |payment_params|
+          OtherIncomePayment.new(category: other_income[:source],
```

Comment:
I think main comment is that you have made category a string, whereas in other PRS the category is a symbol